src/main/java/at/hannibal2/skyhanni/config/features/gui/customscoreboard/AlignmentConfig.java

```diff
@@ -0,0 +1,17 @@
+package at.hannibal2.skyhanni.config.features.gui.customscoreboard;
+
+import com.google.gson.annotations.Expose;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorBoolean;
+import io.github.moulberry.moulconfig.annotations.ConfigOption;
+
+public class AlignmentConfig {
+    @Expose
+    @ConfigOption(name = "Align to the right", desc = "Align the scoreboard to the right side of the screen.")
+    @ConfigEditorBoolean
+    public boolean alignRight = true;
+
+    @Expose
+    @ConfigOption(name = "Align to the center vertically", desc = "Align the scoreboard to the center of the screen vertically.")
+    @ConfigEditorBoolean
+    public boolean alignCenterVertically = true;
+}
```

Comment:
Auto disable these, when the user tries to move the element

Reply:
Are you suggesting changes to your own pr? :O

Reply:
Hey I'm just writing your idea down so I don't forget it

Reply:
implemented